Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.22 AS builder
+FROM golang:1.22.4 AS builder
```

Comment:
just keeping it `1.22` will be better I feel because this always will point to the latest version 1.22 which is 1.22.4 now and 1.22.5 alter when update is available..

Reply:
how did it merge without approving it? 🤔 

cc @Rajalakshmi-Girish

Reply:
> just keeping it 1.22 will be better I feel because this always will point to the latest version 1.22 which is 1.22.4 now and 1.22.5 alter when update is available..

Ok, I will update it next PR.

Reply:
> how did it merge without approving it? 🤔
> 
> cc @Rajalakshmi-Girish

@mkumatag may be because you are the only one in approvers and reviewers list?
I check the logs but didn't find any straightforward answer.

Reply:
Also see https://www.kubernetes.dev/docs/guide/owners/#quirks-of-the-process
`An approver’s /lgtm is simultaneously interpreted as an /approve `